src/routes/(app)/components/CreateLink/graphql/LinkCreate.gql

```diff
@@ -0,0 +1,13 @@
+mutation linkCreate($input: LinkCreateInput!) {
+  linkCreate(input: $input) {
+    link {
+      id
+      originalUrl
+      createdAt
```

Comment:
I recommend using a fragment for cases like this, so we have a type generated for validations

Reply:
You can explain more? I think that I used this fragment

Reply:
I understand, sure!